demoScripts/createVMAndSSH.md

```diff
@@ -0,0 +1,82 @@
+# Create an Azure Linux VM and SSH Into it
+
+## Define Environment Variables
+
+The First step in this tutorial is to define environment variables 
+
+```bash
+export MY_RESOURCE_GROUP_NAME=myResourceGroup
```

Comment:
Meaningful names please, for both the environment variable and the value, e.g.

```
export QUICKSTART_VM_RESOURCE_GROUP=vmWishSSHQuickstart
```

Reply:
Additional note here, having consistent naming convention for env variables is very important. We should ensure that we should make this a part of the documentation standards.